web/src/main/java/org/cbioportal/genome_nexus/GenomeNexusImporter.java

```diff
@@ -21,6 +21,7 @@
     final static String[][] IMPORT_RESOURCE_FILES = new String[][]{
         new String[]{"/ensembl_biomart_transcripts.txt", EnsemblRepositoryImpl.TRANSCRIPTS_COLLECTION},
         new String[]{"/ensembl_biomart_canonical_transcripts_per_hgnc.txt", EnsemblRepositoryImpl.CANONICAL_TRANSCRIPTS_COLLECTION},
+        new String[]{"/pfamA.txt", "pfam.domain"},
```

Comment:
@onursumer why do you prefer here not to use constants?

Reply:
Actually, I don't prefer it :) I just couldn't find the right place to add this constant. We don't have a `PfamDomainRepositoryImpl`, I might have put the constant in the actual interface `PfamDomainRepository`. But then if I want to use the same constant in the model class, it would add a dependency to the `persistence` module in the `model` module, which we want to avoid. 

Same thing is also true for the constant `EnsemblRepositoryImpl.TRANSCRIPTS_COLLECTION`, we include it here but not in the model class. This is because our persistence annotations are still in the `model` module. Ideally we should find a way to add persistence annotations in the `persistence` module, not in the `model` module. But I wasn't able to find an easy way to do it, something similar to our `mixin`s in the web and service modules would be ideal.

Reply:
ok, cool we can leave it for now